mlflow/pyfunc/scoring_server/client.py

```diff
@@ -36,7 +49,10 @@ def invoke(self, data):
         Invoke inference on input data. The input data must be pandas dataframe or json instance.
         """
         content_type = scoring_server.CONTENT_TYPE_JSON
-        post_data = json.dumps(scoring_server._get_jsonable_obj(data, pandas_orient="split"))
+        post_data = json.dumps(
+            scoring_server._get_jsonable_obj(data, pandas_orient="split"),
+            cls=DateTimeEncoder,
+        )
```

Comment:
Could you add a unit test to test function `_dataframe_from_json` works correctly on JSON data including datatime fields ?

Reply:
We'd better add similar unit test for all of datetime.datetime, datetime.date, datetime.time, pd.Timestamp.

Reply:
> Could you add a unit test to test function _dataframe_from_json works correctly on JSON data including datatime fields ?

Added.

Reply:
@WeichenXu123 

How does data contain `datetime.datetime`, `datetime.date`, `datetime.time`, `pd.Timestamp, or `datetime.timedelta`?